packages/rocketchat-ui-sidenav/client/roomList.js

```diff
@@ -73,6 +73,15 @@ Template.roomList.helpers({
 	}
 });
 
+Template.roomList.rendered = function() {
+	if (this.data.identifier !== undefined && this.data.identifier === 'c') {
```

Comment:
This needs to be optional.  Not everyone wants this.  Some want to make use of the home page to show off news and other items.  Also perhaps it could let an admin choose to either take them to a specific channel, or the first channel in the list.